FR #1488: Add Line-Item Examples for Non-Negotiated Commitment Scenarios (#1877)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/appendix/commitment_discounts/aws_overflow_150pct.md

Reviewer: Need to fix PDF build.  Current error:

> The following files: ['gcp_flex_cud_all_upfront.md', 'azure_savings_plan_all_upfront.md', 'gcp_resource_cud_all_upfront.md', 'aws_savings_plan_partial_upfront.md', 'aws_savings_plan_no_upfront.md', 'aws_savings_plan_all_upfront.md', 'azure_reservation_all_upfront.md', 'azure_reservation_no_upfront.md', 'aws_underutilized_75pct.md', 'gcp_flex_cud_partial_upfront.md', 'aws_overflow_150pct.md', 'aws_underutilized_50pct.md', 'aws_reserved_instance_all_upfront.md', 'aws_reserved_instance_partial_upfront.md', 'aws_zero_utilization.md'] are missing from the template appendix/commitment_discounts/commitment_discounts.mdpp

# Amazon Web Services EC2 Instance Savings Plan (All-Upfront)

> **Scenario Type:** commitment

Reviewer 2: May be worth putting these details into a table for readability. Hard to grok the details as bolded narrative (at least for me).

Author: @Matt-Cowsert agreed - make this change in all files is your recommendation?

**Payment Type:** All-Upfront
**Category:** Spend-based
**Utilization:** 150%

> **Hours Generated:** 24

Reviewer 2: ```suggestion
**Hours Generated:** 48 (24 committed + 24 overflow to on-demand)
```
The top-level details show "Hours Generated: 24" but the row summary shows 24 commitment hours + 24 on-demand hours = 48 total usage hours. This could confuse practitioners.

# Amazon Web Services EC2 Instance Savings Plan (All-Upfront)

| Parameter         | Value                                        |
| ----------------- | -------------------------------------------- |
| Scenario Type     | commitment                                   |
| Payment Type      | All-Upfront                                  |
| Category          | Spend-based                                  |

> | Utilization       | 150%                                         |

Reviewer 2: ```suggestion
| Utilization       | 150% (100% committed + 50% overflow to on-demand) |

```
"150% utilization" may confuse practitioners who think of utilization as maxing out at 100%. The term "overflow" is used in the title but not connected to the utilization metric.

| Hours Generated   | 48 (24 committed + 24 overflow to on-demand) |
| Annual Commitment | &dollar;212,000.00                           |
| Committed Rate    | &dollar;24.20/hour                           |
| On-Demand Rate    | &dollar;36.30/hour                           |
| Savings           | 33%                                          |

[CSV Example](/specification/data/commitment_discount_scenarios/aws_overflow_150pct.csv)

## Scenario Description

This example shows a **Amazon Web Services EC2 Instance Savings Plan** (Savings Plan), which is a spend-based commitment where you commit to a specific dollar amount of usage per hour.

The **All-Upfront** payment option means the entire commitment cost is paid at purchase time. This results in a single Purchase row with the full BilledCost and EffectiveCost=0 (since the cost is amortized to usage rows).

> This scenario demonstrates **overflow** at 150% utilization where demand exceeds commitment capacity. The first 24 hours are covered by the commitment (CommitmentDiscountStatus='Used'), while the additional 12 hours spill over to on-demand pricing. On-demand rows have no CommitmentDiscountStatus, PricingCategory='Standard', and BilledCost=EffectiveCost at the full list price.

Reviewer 2: ```suggestion
This scenario demonstrates **overflow** at 150% utilization where demand exceeds commitment capacity by 50%. The first 24 hours represent 100% utilization of the commitment, while the additional 12 hours represent 50% overflow that spills to on-demand pricing. On-demand rows have no CommitmentDiscountStatus, PricingCategory='Standard', and BilledCost=EffectiveCost at the full list price.
```
